### DISCUSSION TITLE
Why can't I use the syntax tailwindcss firt:hidden in Ark?

### DISCUSSION THREAD

**Discussion by wizardAEI** [chakra-ui/ark#1773] [Q&A]
I use the Checkbox and want to control the child by </Checkbox.Control> . but I found that `first:hidden` doesn't work :

```
<Checkbox.Control class='first:hidden'>
              <div>✓</div>
  </Checkbox.Control> 
```
I don't know how to make first:hidden work. Can someone help me solve this? I would be very grateful.

**Comment by cschroeter** [comment]
@wizardAEI 

I have no idea. If you could provide an example reposiotry or an Stackblitz playground I will have a closer look.
